yarn.lock

```diff
@@ -20144,9 +20144,9 @@ ursa-optional@~0.10.0:
     bindings "^1.5.0"
     nan "^2.14.2"
 
-"use-inside@git+https://github.com/aragon/use-inside.git#16f321e499d8f89e9b0fb0999e2d21db2b119bbf":
+"use-inside@https://github.com/aragon/use-inside#16f321e499d8f89e9b0fb0999e2d21db2b119bbf":
```

Comment:
curious: why it automatic changed? Updated the yarn last version? I have 1.22.5

Reply:
I dont know either, I didn't change anything. Just removed `node_modules` and run `yarn`

Reply:
Yah i noticied it two times in local, well i think its not issue, i'll check it later